DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Commands/NoteCommands.cs

```diff
@@ -18,5 +20,22 @@ public NoteCommands(NoteContext notesContext)
             _notesContext.Notes.Add(noteEntity);
             await _notesContext.SaveChangesAsync();
         }
+
+        public async Task<bool> DeleteNote(string userId, string collaboratorId)
+        {
+            var note = _notesContext.Notes.SingleOrDefault(n => n.UserId == userId 
```

Comment:
I know but anyway. If user was deleted from DB that concurrency exception will be raised and you can handled it in another way. So this code obsolete.